libs/ui-lib/lib/ocm/services/types.ts

```diff
@@ -7,7 +7,7 @@ import {
 
 export type ClusterDetailsUpdateParams = Pick<
```

Comment:
(Not for this line)
Based on Nir's comment that OCI should be treated equal to Nutanix and Vsphere, I think `SupportedPlatformIntegrations` needs to include OCI.

Reply:
Changed.